### PR TITLE
Add default rescheduler policy

### PR DESCRIPTION
**Pull request by jimmyxian**
Implement this : #2007
Signed-off-by: Xian Chaobo xianchaobo@huawei.com

### CODE REVIEW

File: cluster/config.go [hunk 1]
```diff
@@ -226,6 +230,18 @@ func (c *ContainerConfig) HasReschedulePolicy(p string) bool {
 	return false
 }
 
+// AddReschdedulePolicy
+func (c *ContainerConfig) AddReschdedulePolicy(policy string) error {
```
Comment (abronan) [inline cluster/config.go]: typo `AddReschdedulePolicy` -> `AddReschedulePolicy`

in comment and function

**Comment by abronan** [comment]
@jimmyxian golint is complaining :ghost: 

Otherwise SGTM, this probably requires some changes in the [rescheduling docs](https://github.com/docker/swarm/blob/master/docs/scheduler/rescheduling.md) as well to mention the added feature.

CI failures seems unrelated.

/cc @vieux

File: cluster/config.go (hunk 1 above)
Comment (jimmyxian) [inline cluster/config.go]: good catch! Thanks @abronan

**Comment by jimmyxian** [comment]
@abronan OK, will add some docs.

**Comment by jimmyxian** [comment]
ping @abronan  @vieux
Review again, thanks

**Comment by jimmyxian** [comment]
ping @abronan  @vieux
Review again, thanks

**Comment by nishanttotla** [comment]
Closing this PR since it's outdated.

**Pull request closed without merging**